src/Components/PresentationalComponents/Snippets/SnippetWithPopover.js

```diff
@@ -71,7 +73,7 @@ const SnippetWithPopover = props => {
                 </StyledText>
             </StackItem>
             <StackItem>
-                {STATUS_OPTIONS.find(option => option.value === cveStatusId.toString()).label || '--'}
```

Comment:
This fallback would throw an error if there was an invalid status_id because it tried to access property label of undefined so I split it into 2 statements.